src/System.Console/src/System/Console.cs

```diff
@@ -119,7 +119,7 @@ private static TextWriter CreateOutputWriter(Stream outputStream)
                 StreamWriter.Null :
                 new StreamWriter(
                     stream: outputStream,
-                    encoding: new ConsoleEncoding(OutputEncoding), // This ensures no prefix is written to the stream.
+                    encoding: OutputEncoding.RemovePreamble(), // This ensures no prefix is written to the stream.
```

Comment:
Why is this RemovePreamble needed?  OutputEncoding calls GetConsoleEncoding which uses RemovePreamble.